### PR TITLE
Podspec linting

### PR DESCRIPTION
**Pull request by RMcNeely**
If fine, then change the version tag and merge into master

### CODE REVIEW

File: ChartIQ.podspec [hunk 1]
```diff
@@ -11,10 +11,10 @@ Pod::Spec.new do |s|
   s.homepage         = 'https://github.com/ChartIQ/Charting-Library---iOS-SDK'
   s.license          = { :type => 'MIT', :file => 'LICENSE' }
   s.author           = { 'jacob' => 'jacob@chartiq.com' }
-  s.source           = { :git => 'https://github.com/ChartIQ/Charting-Library---iOS-SDK.git', :tag => s.version.to_s }
+  s.source           = { :git => 'https://github.com/ChartIQ/Charting-Library---iOS-SDK.git', :branch => 'podspec-linting' }
```
Comment (jdrichards50) [inline ChartIQ.podspec:14]: When do we change the branch it's point to back to what it used to be?

File: ChartIQ.podspec [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'ChartIQ'
-  s.version          = '2.0.2'
+  s.version          = '2.0.3-beta'
```
Comment (jdrichards50) [inline ChartIQ.podspec:3]: So what makes this release a beta?

**Comment by RMcNeely** [comment]
Ideally we merge this into master and then change the source and push up a tag for 2.0.3. Its marked as a beta b/c there is no 2.0.3 tag yet

**Comment by jdrichards50** [comment]
Alright, just as long as we change the pod spec again.

**Comment by RMcNeely** [comment]
Yeah with all the other changes to fix all the linting errors I just wanted a quick review

**Review by jdrichards50 — APPROVED** [review]

**Pull request merged**